src/iai_bullet_sim/utils.py

```diff
@@ -166,29 +176,45 @@ def abs_urdf_paths(file_path, temp_dir):
 
 def abs_sdf_paths(file_path, temp_dir):
     abs_path = Path(res_sdf_model_path(file_path)).absolute()
-    
+
     if hash(abs_path) in _RESOLVED_FILES:
         return _RESOLVED_FILES[hash(abs_path)]
 
     hex_name = md5(str(abs_path).encode('utf-8')).hexdigest()
     temp_file_name = f'{temp_dir}/{hex_name}.sdf'
 
-    with open(file_path, 'r') as of:
-        with open(temp_file_name, 'w') as tf:
-            for l in of:
-                idx = l.find('model://', 0)
-                while idx != -1:
-                    e_idx = l.find('</uri>', idx)
-                    pkg_path = l[idx:e_idx]
-                    r_path = res_sdf_model_path(pkg_path)
-                    # Recursively resolve SDF files
-                    if r_path[-4:].lower() == '.sdf':
-                        r_path = abs_sdf_paths(r_path, temp_dir)
-
-                    l = l.replace(l[idx:e_idx], r_path)
-                    idx = l.find('model://', idx + len(r_path))
-                tf.write(l)
-    
+    with open(abs_path, 'r') as of:
+        tree = ET.parse(of)
+
+    for link in tree.iterfind('.//link'):
+        inertial = link.find('inertial')
+        if inertial is None:
+            inertial = ET.SubElement(link, 'inertial')
+            mass = ET.SubElement(inertial, 'mass')
+            mass.text = str(0)
```

Comment:
Is assuming 0 mass a good idea? That makes everything static.

Reply:
And I think in this case, if `inertial` component is missing, we assume that the object is static. But I still changed it to small default value. Please take a look at my new commit